DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -421,14 +421,13 @@ final class AddressBarTextField: NSTextField {
         }
 #endif
 
-        if DefaultSubscriptionFeatureAvailability().isFeatureAvailable {
```

Comment:
`DefaultSubscriptionFeatureAvailability().isFeatureAvailable` and all calls are removed as it is always `true` post-launch